sfe updates (#146)
codatio/codat-docs: docs/sync-for-expenses/configandcategorize.md

## Configuration

The [configuration endpoint](/sync-for-expenses-api#/operations/get-company-configuration) enables you to set up how your customers expenses will be pushed. 
You can check the config at any time to confirm your companies' configuration.

```http title="Company Config"
GET https://expensesync.codat.io/companies/{companyId}/config
{
    "bankAccount": {
        "id": "{selectedBankAccountId}"
    },
    "supplier": {
        "id": "{selectedSupplierId}"
    },
    "customer": {
        "id": "{selectedCustomerId}"
    }
}
```

Using this endpoint you will be able to:
#### Bank Account
<ul>
   <li>

>       Choose which bank account (<code>bankAccount.id</code>) purchases will be made from, this can also be a credit card account.

Reviewer: ```suggestion
      Choose which bank account (<code>bankAccount.id</code>) purchases will be made from. This can also be a credit card account.
```

   </li>
   <ul>
      <li>

>          <a href="/accounting-api#/operations/get-account">GET</a> A list of available accounts, You can use this request to retrieve a list of relevant accounts from your customers accounting software you should also add additional query parameters e.g. <code>query=metadata.isDeleted=false&&isBankAccount=true</code>. 

Reviewer: ```suggestion
         <a href="/accounting-api#/operations/get-account">GET</a> a list of available accounts.  
         Use this request to retrieve a list of relevant accounts from your customers' accounting software.  You should also add additional query parameters, e.g. <code>query=metadata.isDeleted=false&&isBankAccount=true</code>. 
```

<li>For <strong>credit cards</strong> you can use an additional query parameter e.g. <code>query=metadata.isDeleted=false&&isBankAccount=true&&type=liability</code></li>
      </li>
      <li>

>          You can <a href="/accounting-api#/operations/post-account">POST</a> to the accounting api to <strong>create</strong> a new account.

Reviewer: ```suggestion
         You can <a href="/accounting-api#/operations/post-account">POST</a> to the Accounting API to <strong>create</strong> a new account.
```

Sync for Expenses currently supports option 2 only.
:::

#### Supplier
<ul>
   <li>

>       A (<code>supplier.id</code>) is required to associate all spend against. 

Reviewer: ```suggestion
      A supplier (<code>supplier.id</code>) is required to associate all spending against that supplier. 
```

   </li>
   <ul>
      <li>

>          <a href="/accounting-api#/operations/get-supplier">GET</a> A list of available suppliers in the companies accounting software, you can also add additional query parameters e.g. <code>query=metadata.isDeleted=false&&supplierName=supplierName</code>

Reviewer: ```suggestion
         <a href="/accounting-api#/operations/get-supplier">GET</a> a list of available suppliers in the company's accounting software. You can also add additional query parameters, e.g. <code>query=metadata.isDeleted=false&&supplierName=supplierName</code>
```

      </li>
      <li>
         You can <a href="/accounting-api#/operations/post-supplier">POST</a> to create a new supplier.
      </li>
   </ul>
</ul>

#### Customer
<ul>
   <li>

>       Choose which customer (<code>bankAccount.id</code>) income related activities such as cashback will be associated too.  

Reviewer: ```suggestion
      Choose which customer (<code>customer.id</code>) any income-related activities, such as cashback, should be associated with.  
```
